Automattic/wp-calypso
# Remove pattern picker package and usage

Related to #

## Proposed Changes

* Removing the pattern-picker package since it no longer used.
The pattern-picker has been merged into the design-carousel package

## Testing Instructions



* Checkout this branch
* Verify there are no references to the pattern-picker package
* Verify there are no console errors on calypso

## Pre-merge Checklist
